src/jest-setup.ts

```diff
@@ -3,9 +3,23 @@ import { reset } from './mockedEndpointsService';
 import { allowRealNetwork } from './settings';
 import { overrideFetch } from './overrideFetch';
 
+// make sure that axios is a singleton
+let actualAxios: any;
+jest.doMock('axios', () => {
```

Comment:
this is some magic stuff, it makes sure that we don't get two different axios instances because of import/require differences.